src/window.rs

```diff
@@ -140,6 +142,8 @@ impl WindowWrapper {
             .expect("Failed to create window");
         info!("window created");
 
+        let window = Sdl2Window::new(&sdl_window);
```

Comment:
Maybe grouping window and skulpin_renderer in a local scope which returns the created window would help to show that the sdl2window isn't used again and is immediately dropped? I believe the compiler is doing that under the covers or else a borrow error would have showed up at creation of the WindowWrapper.